### DISCUSSION TITLE
Menu Entry Swapper name caan't be found

### DISCUSSION THREAD

**Discussion by AGuyWhoLikesCheese666** [runelite/runelite#16244] [Q&A] [answered]
I'm typing in the name ' Menu Entry Swapper ' but it doesn't popup in the plugins menu, is it named something else?

**Comment by Felanbird** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/41973452/216677834-66e97256-4fd2-495b-99d6-d577c0f3384a.png)

**Reply by Nightfirecat** [reply]
Make sure you're searching your installed plugins, and not searching in the plugin hub.
